kubernetes_env/benchmark/benchmark.py

```diff
@@ -143,22 +169,35 @@ def start_benchmark(fortio, filter_dirs, platform, threads, qps, time):
                       " Make sure you give the right path")
             return util.EXIT_FAILURE
 
-        log.info("Running fortio...")
-        # Might break for filter_dir/
-        fname = fd.split("/")[-1]
+        # wait for kubernetes set up to finish
+        time.sleep(10)
+        fname = Path(fd).name
+        filters.append(fname)
+        log.info("Warming up...")
+        warmup_res = do_burst(product_url, platform, threads, 100, 10)
```

Comment:
Honestly in terms of warmup I believe a couple requests is good enough. You do not need to send that many.